pegasus/helpers/section_api_helpers.rb

```diff
@@ -107,10 +107,10 @@ def self.update_if_allowed(params, dashboard_user_id)
     user_to_update = Dashboard.db[:users].where(id: params[:id], deleted_at: nil)
     return if user_to_update.empty?
     return if Dashboard.db[:sections].
-      join(:followers, section_id: :sections__id).
-      where(sections__user_id: dashboard_user_id, sections__deleted_at: nil).
-      where(followers__student_user_id: params[:id], followers__deleted_at: nil).
-      empty?
+        join(:followers, section_id: :sections__id).
+        where(sections__user_id: dashboard_user_id, sections__deleted_at: nil).
+        where(followers__student_user_id: params[:id], followers__deleted_at: nil).
+        empty?
```

Comment:
OIC. That makes sense. This does seem consistent with the double indent under if or unless.  I didn't notice that before.